frontend/src/features/videos/VideoCard.tsx

```diff
@@ -125,6 +127,39 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
+const PlayerWrapper = ({
+  duration,
+  children,
+}: {
+  duration?: string;
+  children: React.ReactNode;
+}) => {
+  const [isDurationVisible, setIsDurationVisible] = useState(true);
+  return (
+    <Box
+      position="relative"
+      height="100%"
+      onClick={() => setIsDurationVisible(false)}
+    >
+      {isDurationVisible && duration && (
+        <Box
+          position="absolute"
+          bottom={0}
+          right={0}
+          bgcolor="rgba(0,0,0,0.5)"
+          color="#fff"
+          px={1}
+          fontSize="0.8em"
+          fontWeight="bold"
```

Comment:
A small bug: the duration block will disappear if you click on it. We would probably prefer that pointer events pass through this block.

```suggestion
          fontWeight="bold"
          style={{ pointerEvents: 'none' }}
```